### PR TITLE
fix: Add Solution in Curriculum/Challenges/English/08/CodingInterviewProblems/DataStructures/CREATE_LINKED_LIST_CLASS.md

### PR DESCRIPTION
**Pull request by StanimalTheMan**
- [x ] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [ x] My pull request targets the `master` branch of freeCodeCamp.
- [ x] None of my changes are plagiarized from another source without proper attribution.
- [ x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x ] My changes do not use shortened URLs or affiliate links.

If what I am doing in the curriculum directory is ok, I will also try to fix the guide directory for this problem as well that is where the stub is and I can see what the change will look like.  I updated the guide directory previously for the AdjacencyList challenge and the pull request was merged not too long ago.  Thank you.

### CODE REVIEW

**Comment by StanimalTheMan** [comment]
@thecodingaviator @RandellDawson Are fixes like this unnecessary?  Thank you.

**Comment by thecodingaviator** [comment]
@StanimalTheMan these foxes are fine until the solutions are alright and someone else has not already taken it up

File: curriculum/challenges/english/08-coding-interview-prep/data-structures/create-a-linked-list-class.english.md [hunk 1]
```diff
@@ -78,6 +78,38 @@ function LinkedList() {
...
+    return length;
+  };
+
+  this.add = function(element){
+    // Only change code below this line
+    if (head == null) {
+      head = new Node(element);
+    } 
+    else {
+      let headCopy = head;
```
Comment (RandellDawson) [inline curriculum/challenges/english/08-coding-interview-prep/data-structures/create-a-linked-list-class.english.md]:
```suggestion
      let currentNode = head;
```

File: curriculum/challenges/english/08-coding-interview-prep/data-structures/create-a-linked-list-class.english.md [hunk 2]
```diff
@@ -78,6 +78,38 @@ function LinkedList() {
...
+      while (headCopy.next != null) {
```
Comment (RandellDawson) [inline curriculum/challenges/english/08-coding-interview-prep/data-structures/create-a-linked-list-class.english.md]:
```suggestion
      while (currentNode.next != null) {
```

File: curriculum/challenges/english/08-coding-interview-prep/data-structures/create-a-linked-list-class.english.md [hunk 3]
```diff
@@ -78,6 +78,38 @@ function LinkedList() {
...
+        headCopy = headCopy.next; //headCopy.next is now the last node of linked list
```
Comment (RandellDawson) [inline curriculum/challenges/english/08-coding-interview-prep/data-structures/create-a-linked-list-class.english.md]:
```suggestion
       currentNode = currentNode.next; // currentNode.next will be last node of linked list after loop
```

File: curriculum/challenges/english/08-coding-interview-prep/data-structures/create-a-linked-list-class.english.md [hunk 4]
```diff
@@ -78,6 +78,38 @@ function LinkedList() {
...
+      }
+      headCopy.next = new Node(element);
```
Comment (RandellDawson) [inline curriculum/challenges/english/08-coding-interview-prep/data-structures/create-a-linked-list-class.english.md]:
```suggestion
      currentNode.next = new Node(element);
```

**Review by RandellDawson — DISMISSED** [review]
@StanimalTheMan While this solution does meet the challenge requirements and passes all test, I think renaming the variable makes it a bit more readable.

**Comment by RandellDawson** [comment]
@StanimalTheMan Can you please make the changes requested?

**Comment by RandellDawson** [comment]
I have made my suggested changed on behalf of the user, since the user had a valid solution but I felt the variable names should be similar to what is used in some of the other linked list solutions.

**Review by RandellDawson — APPROVED** [review]
Looks good now.

**Review by ojeytonwilliams — APPROVED** [review]
LGTM :+1:

**Review by moT01 — APPROVED** [review]
LGTM @StanimalTheMan - thanks for contributing to freeCodeCamp! 🎉 

Thanks for commiting changes @RandellDawson 🎉 

There must not have been an issue related to this?

**Pull request merged**